tests/installation/change_desktop.pm

```diff
@@ -20,33 +20,67 @@ sub change_desktop() {
     my ($self) = @_;
     # ncurses offers a faster way
     if (check_var('VIDEOMODE', 'text')) {
-        send_key 'alt-c';
+
+        wait_screen_change {
+            send_key 'alt-c';
+        };
+
```

Comment:
That's a bit verbose. Please use just `wait_screen_change { send_key 'alt-c' };`

Also the synchronization is only required where we don't have one already afterwards. Here the next call will be `assert_screen` so isotovideo will wait anyway for the screen. No need to put the additional `wait_screen_change`